recipes/Dessert/frozen_chocolate_cake.md

```diff
@@ -0,0 +1,38 @@
+# Frozen Chocolate Cake
+(not frikin pudding)
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
```

Comment:
Errr

> 10. After that keep it in the fridge for atleast 2 hours before serving.  

I think giving this a 10 min speed time is misleading. Maybe it could have a special prep time badge